docker-compose.yml

```diff
@@ -7,7 +7,6 @@ services:
       dockerfile: ./docker/images/backend/Dockerfile
       target: moodlenet
       args:
-        BACKEND_VERSION: "master"
```

Comment:
Maybe we can keep this variable with a fallback on master ? Or maybe there is no reason anymore to keep all the `build` config ?

```
BACKEND_VERSION: ${BACKEND_VERSION:-master}
```

Reply:
Hmm yes, I think there is no reason anymore to keep the build config here.
I'll remove it, thanks !